src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,42 @@
+.nav {
+  &__list {
+    display: flex;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: inline-block;
+    padding: 23px 0 ;
+    font-family: inherit;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14px;
+    text-transform: uppercase;
+    cursor: pointer;
+    color: $primary-text-color;
+  }
+
+  &__link:hover {
+    color: $accent-color;
+    transition: 0.3s color;
```

Comment:
move transition to link styles

Reply:
could you explane me, why we can't use this property with pseudo elemet "hover"?